keys/keys.go

```diff
@@ -289,5 +289,11 @@ func DecodeRangeMetaKey(key proto.Key) (proto.Key, proto.Key) {
 	}
 	// Otherwise find the first entry greater than the given key in the same meta prefix.
 	metaPrefix := proto.Key(key[:len(Meta1Prefix)])
+	// Edge case for "\x00\x00meta1\xff\xff", which cannot return
+	// key.Next() as which itself is the last key in Meta1Prefix range.
+	meta1KeyMax := proto.MakeKey(Meta1Prefix, proto.KeyMax)
```

Comment:
1. Not doing this is that other Max in constants.go like SystemMax, MetaMax greater than the concat(SystemPrefix, \xff\xff), so I am not sure this Meta1KeyMax should define like this. 
2 .Current Meta2 is not used in RangeLookup key, it seems no issue.